src/FluentMigrator.Runner.SqlServer/Generators/SqlServer/SqlServer2005Generator.cs

```diff
@@ -228,7 +228,7 @@ public override string Generate(CreateIndexExpression expression)
                 CreateIndex,
                 GetUniqueString(expression),
                 GetClusterTypeString(expression),
-                Quoter.QuoteIndexName(expression.Index.Name),
+                Quoter.QuoteIndexName(expression.Index.Name, expression.Index.SchemaName),
```

Comment:
@emejibka Are you sure this is correct?  In SQL Server, the schema name of the index is inferred by the table's schema.  Indexes don't exist outside of a table schema, so there is no concept of an index name consisting of a name + schema.  I can't speak to Postgres or Oracle or the other DBs, but this is likely wrong for SQL Server.